pynitrokey/cli/nk3/__init__.py

```diff
@@ -892,3 +562,7 @@ def provision_fido2(ctx: Context, key_file: BinaryIO, cert_file: BinaryIO) -> No
         provisioner = ProvisionerApp(device)
         provisioner.write_file(b"fido/x5c/00", cert)
         provisioner.write_file(b"fido/sec/00", key)
+
+
+# This import has to be added here to avoid circular dependency
+from . import secrets  # noqa: F401,E402
```

Comment:
I don’t understand.  AFAIK, the import is not used – why do we need it at all?

Reply:
The subcommand won't get initialized, and won't show up in the click generated menu otherwise. I will expand the comment.
Unless - do you know better solution for that?

Reply:
It imports the subcommand from the `secrets.py` module.

Reply:
I could not improve much here quickly. Leaving as is.